architecture/users/readme.md

```diff
@@ -90,9 +84,9 @@
     ]
     ```
 
-### USR-F3 Я как пользователь хочу иметь возможность получить пользователей по их slack name.
+### USR-F3 I, as a user, want to be able to get list of students on their slack name.
```

Comment:
~I, as a user,~ **As a user I** want ~to be able~ to get list of students ~on~ **by** their slack name.